app/services/validations_test.go

```diff
@@ -1,14 +1,31 @@
-package services_test
+package services
```

Comment:
I realise I am diverging from the house style here. I didn't think the black box tests were providing much value here, but if you like I can alternatively:

1. Introduce an internal test with access to the `passwordValidator` function
2. Export `passwordValidator`
3. Import our zxcvbn library and test it directly
4. Just choose to believe this library's own tests and assertions and omit the test altogether

Reply:
I'm happy with 4 because we can't do better than:

> all unit tests from the upstream library have been ported (and even more tests have been added) to ensure that this holds.

from https://github.com/trustelem/zxcvbn/blob/master/README.md